### DISCUSSION TITLE
bug: Capacitor on existing Nuxt project

### DISCUSSION THREAD

**Discussion by shidcordero** [ionic-team/capacitor#3145] [Q&A]
## Bug Report
I am working on a web project and I want it to be deployed in a native android mobile. I am using NuxtJS and I use `nuxt generate` to generate static files which has the index.html. I followed all the procedures on the web site documentation but I still having issue running the app. My app works fine on web browsers but when running on the mobile using capacitor, it shows this error on the logcat

> 2020-06-24 16:03:58.342 14047-14102/com.onlinepartyroom E/Capacitor: Unable to open asset URL: http://localhost/en-us/_nuxt/runtime.d56a943.js
> 2020-06-24 16:03:58.342 14047-14096/com.onlinepartyroom E/Capacitor: Unable to open asset URL: http://localhost/en-us/_nuxt/commons.app.cc55111.js
> 2020-06-24 16:03:58.343 14047-14102/com.onlinepartyroom E/Capacitor: Unable to open asset URL: http://localhost/en-us/_nuxt/app.994d6a3.css
> 2020-06-24 16:03:58.344 14047-14096/com.onlinepartyroom E/Capacitor: Unable to open asset URL: http://localhost/en-us/_nuxt/vendors.app.822afb8.js

### Capacitor Version

`  Capacitor Doctor   

Latest Dependencies:

  @capacitor/cli: 2.2.0

  @capacitor/core: 2.2.0

  @capacitor/android: 2.2.0

  @capacitor/electron: 2.2.0

  @capacitor/ios: 2.2.0

Installed Dependencies:

  @capacitor/electron not installed

  @capacitor/android 2.2.0

  @capacitor/cli 2.2.0

  @capacitor/core 2.2.0

[success] Android looking great! �`

### Platform(s)
I've only tried android since I don't have XCode

### Current Behavior
Can run the app but it won't show anything

### Expected Behavior

Should run the app

### Code Reproduction
No special code here, just a normal NuxtJS application

### Other Technical Details

`npm --version` output: 6.14.4

`node --version` output: v13.13.0

**Comment by DimosthenisK** [comment]
You probably need to run ``nuxt generate --no-ssr`` instead of ``nuxt generate``

**Comment by shidcordero** [comment]
> You probably need to run `nuxt generate --no-ssr` instead of `nuxt generate`

I already tried that one and still shows same error.

**Comment by shidcordero** [comment]
this is now okay. the error that causes the issue was the route base path. any ideas how to make it work if I have a route base path?

**Comment by jcesarmobile** [comment]
Android Studio doesn't copy folders staring by `_`
See how to make it copy them https://github.com/ionic-team/capacitor/issues/1750#issuecomment-511785165
Or how to make nuxt not create the folder staring with `_` https://github.com/ionic-team/capacitor/issues/3111#issuecomment-644500445
